lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -15,8 +15,8 @@ class FirstMeaningfulPaint extends Audit {
   static get meta() {
     return {
       name: 'first-meaningful-paint',
-      description: 'First meaningful paint',
-      helpText: 'First meaningful paint measures when the primary content of a page is visible. ' +
+      description: 'First Meaningful Paint',
```

Comment:
is it possible to squash/rebase/something off of #5130? definitely confusing having all of that in here when it's going to merge with it

Reply:
rebased. much cleaner.